subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -1,220 +1,192 @@
 [[viewing-debugging-dependencies]]
-= Viewing and debugging dependencies
-
-Gradle provides sufficient tooling to navigate large dependency graphs and mitigate situations that can lead to link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell].
-Users can choose to render the full graph of dependencies as well as identify the selection reason and origin for a dependency.
-The origin of a dependency can be a declared dependency in the build script or a transitive dependency in graph plus their corresponding configuration.
-Gradle offers both capabilities through visual representation via build scans and as command line tooling.
-
 [[sec:debugging-build-scans]]
-== Build scans
+= View and Debug Dependencies
 
-[NOTE]
-====
-If you do not know what link:https://scans.gradle.com/[build scan]s are, be sure to check them out!
-====
+Gradle provides tooling to navigate dependency graphs and mitigate link:https://en.wikipedia.org/wiki/Dependency_hell[dependency hell].
+Users can render the full graph of dependencies as well as identify the selection reason and origin for a dependency.
+Dependencies can originate through build script declared dependencies or transitive dependencies.
+You can visualize dependencies with the built-in Gradle CLI `dependencies` task or with link:https://scans.gradle.com/[build scans].
 
-A build scan can visualize dependencies as a navigable, searchable tree.
-Additional context information can be rendered by clicking on a specific dependency in the graph.
+[[sec:listing_dependencies]]
+== List Project Dependencies
 
-.Dependency tree in a build scan
-image::dependency-management-dependencies-report-build-scan.png[]
```

Comment:
I would keep the build scan example but move it close to the output example for `dependencies`.

Reply:
I feel that leaving the image distracts from the core idea behind the page: explaining the `dependencies` task. Since we link to other documentation that explains the ins and outs of build scans, I think it's OK to lose this example.

I would be comfortable adding the example back in if we elaborated a bit more on how to use the build scan tree for real world debugging use cases. Maybe there's another section of the documentation we could consolidate here? I'll look, but if you know of any, you could save me some time.

Reply:
My comment came from the fact that build scan example for `dependencyInsight` has been preserved.

Like I said, the flow of the page is much better with your changes. So if you think having it lower down the page is too much noise, good for me.